### PR TITLE
Bump to the latest protobuf APIs

### PR DESCRIPTION
**Pull request by eyakubovich**
Also cleans up the output of server-stub by
only printing the inner request.

### CODE REVIEW

File: src/server-stub/main.rs [hunk 1]
```diff
@@ -35,12 +35,28 @@ impl TokenService for Service {
...
     }
 
+    async fn enroll_cluster_agent(
+        &self,
+        request: Request<pb::EnrollClusterAgentRequest>,
+    ) -> Result<Response<pb::EnrollClusterAgentResponse>, Status> {
+
+        println!("enroll_cluster_agent: {:?}", request.into_inner());
+
+        let expiration = SystemTime::now() + Duration::from_secs(180);
```
Comment (crawford) [inline src/server-stub/main.rs:45]: Is there anything special about 3 minutes?

File: src/server-stub/main.rs [hunk 2]
```diff
@@ -35,12 +35,28 @@ impl TokenService for Service {
...
+
+        let reply = pb::EnrollClusterAgentResponse{
+            session_token: "SESSION_TOKEN".to_string(),
```
Comment (crawford) [inline src/server-stub/main.rs:48]: Does the session token need to follow the same format as a real one, or is it treated as an opaque blob everywhere?

**Review by crawford — APPROVED** [review]
Looks good.

File: src/server-stub/main.rs (hunk 1 above)
Comment (eyakubovich) [reply src/server-stub/main.rs:45]: No but when testing, I don't want to wait too long.

File: src/server-stub/main.rs (hunk 2 above)
Comment (eyakubovich) [reply src/server-stub/main.rs:48]: treated as opaque blob. I chose something simple b/c it used to be printed (in this server-stub) and I wanted to see that it looked correct.

**Pull request merged**